ArcPicker/ArcPicker.js

```diff
@@ -0,0 +1,174 @@
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import Arc from '../Arc';
+import ArcPickerBehaviorDecorator from './ArcPickerBehaviourDecorator';
+import Skinnable from '../Skinnable';
```

Comment:
Please distinguish between the components out of the current module and the components in the current module.

```suggestion
import Arc from '../Arc';
import Skinnable from '../Skinnable';

import ArcPickerBehaviorDecorator from './ArcPickerBehaviourDecorator';
```